[2025-01-26T22:15:09.085+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-26T22:15:09.132+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:14:00+00:00 [queued]>
[2025-01-26T22:15:09.153+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:14:00+00:00 [queued]>
[2025-01-26T22:15:09.154+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-26T22:15:09.186+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): stg-load> on 2025-01-26 22:14:00+00:00
[2025-01-26T22:15:09.202+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1937) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-26T22:15:09.205+0000] {standard_task_runner.py:72} INFO - Started process 1939 to run task
[2025-01-26T22:15:09.207+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_gener_data', 'stg-load', 'scheduled__2025-01-26T22:14:00+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/DAG_test_data.py', '--cfg-path', '/tmp/tmpkx_bt4cz']
[2025-01-26T22:15:09.213+0000] {standard_task_runner.py:105} INFO - Job 461: Subtask stg-load
[2025-01-26T22:15:09.350+0000] {task_command.py:467} INFO - Running <TaskInstance: load_gener_data.stg-load scheduled__2025-01-26T22:14:00+00:00 [running]> on host a664bcf281bf
[2025-01-26T22:15:10.018+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_gener_data' AIRFLOW_CTX_TASK_ID='stg-load' AIRFLOW_CTX_EXECUTION_DATE='2025-01-26T22:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-26T22:14:00+00:00'
[2025-01-26T22:15:10.020+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-26T22:15:10.123+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-01-26T22:15:10.201+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-26T22:15:10.234+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-26T22:15:10.235+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_gener_data, task_id=stg-load, run_id=scheduled__2025-01-26T22:14:00+00:00, execution_date=20250126T221400, start_date=20250126T221509, end_date=20250126T221510
[2025-01-26T22:15:10.320+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-26T22:15:10.346+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
